src/main/java/com/avispl/symphony/dal/device/axis/m3064/AxisCommunicator.java

```diff
@@ -952,6 +953,10 @@ private String retrieveResolution() {
 	 * @throws Exception if device set the value fails or is device can't call URL
 	 */
 	private void setControlTextOverlay(String value) {
+		if (!StringUtils.isNullOrEmpty(value) && value.getBytes(StandardCharsets.UTF_8).length > 80) {
```

Comment:
80 should be a constant with appropriate text
it covers for all cases as we discussed: 1byte, 2bytes, 3 bytes ..., right?

Reply:
1. I update the constant. 
2. Yes, It covers all cases and only throw the exception if the text bytes exceeds the limit of 80 bytes
3. Yes, I updated the comment in the ticket